services/director/src/simcore_service_director/producer.py

```diff
@@ -288,6 +291,18 @@ async def _create_docker_service_params(
             if "Limits" in param["value"] or "Reservations" in param["value"]:
                 docker_params["task_template"]["Resources"].update(param["value"])
 
+            # ensure strictness of reservations/limits (e.g. reservations = limits)
+            for key in ["NanoCPUs", "MemoryBytes"]:
+                max_value = max(
+                    docker_params["task_template"]["Resources"]["Reservations"][key],
```

Comment:
can reservation be larger than limits?
Just to be sure. This tries to avoid exactly that but settings bot the the same?

Reply:
so we discussed at the PM2 that we want to have strict resources. meaning they should be the same instead of having a large delta. we can still discuss that if you like.